vendor.yml

```diff
@@ -0,0 +1,72 @@
+---
+js:
+  reg:
+  - assets/js/legend.js
+  - bower_components/angular-multi-select/isteven-multi-select.js
+  - bower_components/bootbox.js/bootbox.js
+  - bower_components/bootstrap/dist/js/bootstrap.js
+  - bower_components/Chart.js/Chart.js
+  - bower_components/filesize/lib/filesize.js
+  - bower_components/jquery/dist/jquery.js
+  - bower_components/lodash/dist/lodash.js
+  - bower_components/moment/moment.js
+  - bower_components/splitargs/src/splitargs.js
+  - bower_components/toastr/toastr.js
+  - bower_components/xterm.js/dist/xterm.js
+  min:
+  - assets/js/legend.js
+  - bower_components/angular-multi-select/isteven-multi-select.js
+  - bower_components/bootbox.js/bootbox.js
+  - bower_components/bootstrap/dist/js/bootstrap.min.js
+  - bower_components/Chart.js/Chart.min.js
+  - bower_components/filesize/lib/filesize.min.js
+  - bower_components/jquery/dist/jquery.min.js
```

Comment:
I've actually tested it with `run-dev` task and as far as I checked the only jquery's position was a problem.